app/views/notification_mailer/new_lecture_email.html.erb

```diff
@@ -1,5 +1,10 @@
-<%= t('mailer.new_lecture', title: @lecture.title_with_teacher) %>
-<br>
-<%= t('mailer.subscribe_lecture_html',
-      profile: link_to(t('notifications.profile'),
-                       edit_profile_url)) %>
\ No newline at end of file
+<%= t('mailer.new_lecture', title: @lecture.title_with_teacher) %>
+<br>
+<%= t('mailer.subscribe_lecture_html',
+      profile: t('notifications.profile')) %>
```

Comment:
This will work but as above I would suggest using
```
<%= t('mailer.subscribe_lecture') %>
```
and setting `mailer.subscribe_lecture` appropriately in the locale files (in particular, without profile parameter).